src/content/app/genome-browser/components/track-settings-panel/track-settings-views/GeneTrackSettings.tsx

```diff
@@ -42,37 +42,31 @@ export const GeneTrackSettings = (
     getTrackSettingsForTrackId(state, trackId)
   )?.settings as GeneTrackSettingsType;
 
-  const {
-    updateTrackName,
-    updateFeatureLabelsVisibility,
-    updateShowSeveralTranscripts,
-    updateShowTranscriptIds
-  } = useTrackSettings({ selectedTrackId: trackId });
+  const { updateTrackSetting } = useTrackSettings({ selectedTrackId: trackId });
 
   if (!selectedTrackSettings) {
     return null;
   }
 
-  const shouldShowTrackName = selectedTrackSettings.showTrackName;
-  const shouldShowFeatureLabels = selectedTrackSettings.showFeatureLabels;
-  const shouldShowSeveralTranscripts =
-    selectedTrackSettings.showSeveralTranscripts;
-  const shouldShowTranscriptIds = selectedTrackSettings.showTranscriptIds;
+  const shouldShowTrackName = selectedTrackSettings.name;
+  const shouldShowFeatureLabels = selectedTrackSettings.label;
+  const shouldShowSeveralTranscripts = selectedTrackSettings.several;
+  const shouldShowTranscriptIds = selectedTrackSettings['transcript-label'];
 
   const handleSeveralTranscriptsToggle = () => {
-    updateShowSeveralTranscripts(!shouldShowSeveralTranscripts);
+    updateTrackSetting('several', !shouldShowSeveralTranscripts);
```

Comment:
I think it would be good if there is a mention of transcripts instead of just `several` like `several-transcripts`

Reply:
I don't disagree; but that should really be the name of the switch on the eard side. Using one set of names in ensembl-client, and a different set of names in eard, will lead to unnecessary confusion and unnecessary code for translating naming convention into another. Meahwhile, I've left [comments in trackSettingsSlice](https://github.com/Ensembl/ensembl-client/blob/f6bbe0321bca92cdefa612283b393ae63d970eb0/src/content/app/genome-browser/state/track-settings/trackSettingsSlice.ts#L32) explaining what these eard keywords mean.